src/components/Header.js

```diff
@@ -109,7 +109,11 @@ class Header extends Component {
               </NavLink>
             </NavItem>
             <NavItem>
-              <NavLink tag={Link} to="/advance">
+              <NavLink
+                className="d-flex justify-content-center"
+                tag={Link}
+                to="/advance"
```

Comment:
Should we rename the url as well now that the menu nav is renamed to `advanced`?